Makefile

```
@@ -9,16 +10,22 @@ DGPU_TYPE ?= arc  # arc|flex
 PIPELINE_COUNT?= 1
 PIPELINE_SCRIPT ?= yolov5s_full.sh
 TARGET_FPS ?= 14.95
+DOCKER_COMPOSE ?= docker-compose.yml
+RETAIL_USE_CASE_ROOT ?= ..
+RESULTS_DIR ?= ../results
 
 download-models:
 	./download_models/downloadModels.sh
 
+download-sample-videos:
+	cd performance-tools/benchmark-scripts && ./download_sample_videos.sh
+
 clean-models:
 	@find ./models/ -mindepth 1 -maxdepth 1 -type d -exec sudo rm -r {} \;
 
-run-smoke-tests:
+run-smoke-tests: download-models update-submodules download-sample-videos
```

suggesting make this dependencies run in serial in case some crazy one run make in parallel mode:
```suggestion
run-smoke-tests: | download-models update-submodules download-sample-videos
```